db_provision.mjs

```diff
@@ -0,0 +1,165 @@
+import { db } from "@vercel/postgres";
+import * as dotenv from "dotenv";
+dotenv.config({
+  path: ".env.development.local",
+});
+
+const client = await db.connect();
+
+// createProductsTable();
+fetchAndInsertProducts("mask");
+// createProductImagesTable();
+
+async function insertProduct(id, brand_id, name, description, ingredients) {
```

Comment:
Let's re-order functions. First the `fetchAndInsertProducts` one. Then the ones that individually insert items, and last the `create` ones.